### ISSUE TITLE
MultiGrid accessibility error

### ISSUE THREAD

**Issue by pavel-bublavy** [bvaughn/react-virtualized#1540]
MultiGrid component accessibility audit fails with following error:

Elements with an ARIA [role] that require children to contain a specific [role] are missing some or all of those required children.

Some ARIA parent roles must contain specific child roles to perform their intended accessibility functions. 

Failing Elements
`<div class="ReactVirtualized__Grid__innerScrollContainer" role="rowgroup" style="width: 504px; height: 32px; max-width: 504px; max-height: 32px; overflow: hidden; position: relative;">`

```
div.ReactVirtualized__Grid__innerScrollContainer
div.ReactVirtualized__Grid__innerScrollContainer
div.ReactVirtualized__Grid__innerScrollContainer
```

The reason of the error is not fulfilling WAI-ARIA rules. Role "rowgroup" requires children of role "row", and role "row" requires children of role "columnheader", "rowheader", or "gridcell".

The roles for "columnheader", "rowheader", or "gridcell" can be assigned to cells in cellRenderer but 
there is currently no way to add element with role "row" which is necessary as aria context for those roles.

Possible solution to this problem should be wrapping grid cells on the same row in another element with role "row" in MultiGrid component implementation.

**Comment by MihirGH** [comment]
Hi @pavel-bublavy 
Did you find any workaround to fix this?
